content/security/advisory/2017-02-01.adoc

```diff
@@ -45,10 +45,10 @@ Users with the permission to configure jobs were able to inject JavaScript into
 Jenkins bundled an outdated version of jbcrypt that was affected by CVE-2015-0886.
 
 
-=== Pipeline metadata files not blacklisted in agent-to-master security subsystem
+=== Pipeline metadata files not excluded in agent-to-master security subsystem
```

Comment:
This reads like a mess, but on reflection, the old version wasn't much clearer either. 😅